docs/src/misc.rst

```diff
@@ -566,6 +578,35 @@ API
 
     .. versionadded:: 1.9.0
 
+.. c:function:: int uv_os_get_passwd2(uv_passwd_t* pwd, uv_uid_t uid)
+
+    Gets a subset of the password file entry for the provided uid.
+    The populated data includes the username, euid, gid, shell,
+    and home directory. On non-Windows systems, all data comes from
+    :man:`getpwuid_r(3)`. On Windows, uid and gid are set to -1 and have no
+    meaning, and shell is `NULL`. After successfully calling this function, the
+    memory allocated to `pwd` needs to be freed with
+    :c:func:`uv_os_free_passwd`.
+
+    .. versionadded:: 1.45.0
+
+.. c:function:: int uv_os_get_group(uv_group_s* group, uv_uid_t gid)
+
+    Gets a subset of the group file entry for the provided uid.
```

Comment:
Either the parameter name is a typo or the description has a typo (they both should be `gid` or both should be `uid`)

Reply:
You are right. It comes directly from the definition.

https://github.com/libuv/libuv/blob/541329d51f82bce83bd81ff7640edabfc32d149c/src/unix/core.c#L1274

It has the gid struct defined but it is not used.

https://github.com/libuv/libuv/blob/v1.x/include/uv/unix.h#L165

Changing a signature for a landed and release API is a breaking change, not sure what any others collaborators think about that.

Reply:
It _is_ an API change but it's probably fine. Are there systems where gid_t and uid_t are different types? They're definitely the same on linux and freebsd.

(In hindsight, we maybe should've defined both uv_gid_t and uv_uid_t as typedefs for id_t, then that question wouldn't have even come up, but oh well, water under the bridge.)